snapcraft/storeapi/constants.py

```diff
@@ -16,8 +16,12 @@
 from __future__ import absolute_import, unicode_literals
 
 
+# FIXME: snapcraft targets the '16' series, hardcode it until more choices
+# become available server side -- vila 2016-04-22
+DEFAULT_SERIES = '16'
 SCAN_STATUS_POLL_DELAY = 5
 SCAN_STATUS_POLL_RETRIES = 5
 UBUNTU_SSO_API_ROOT_URL = 'https://login.ubuntu.com/api/v2/'
 UBUNTU_STORE_API_ROOT_URL = 'https://myapps.developer.ubuntu.com/dev/api/'
```

Comment:
dev stands for devportal, I think it goes back to the days where you were in the u1 team and I wasn't. I don't know why the API is not versioned. I would guess because, whether or not it's versioned, it's supported and versioning other APIs didn't prove to be that useful to maintain backwards compatibility.